src/pages/account/center/index.tsx

```diff
@@ -0,0 +1,247 @@
+    if (tabKey === 'articles') {
+      return <Articles />;
+    }
+    return null;
+  };
+
+  render() {
+    const { newTags, inputVisible, inputValue, tabKey } = this.state;
+    const { currentUser, currentUserLoading } = this.props;
+    const dataLoading = currentUserLoading || !(currentUser && Object.keys(currentUser).length);
+    return (
+      <GridContent>
+        <Row gutter={24}>
+          <Col lg={7} md={24}>
+            <Card bordered={false} style={{ marginBottom: 24 }} loading={dataLoading}>
+              {!dataLoading && (
+                <div>
+                  <div className={styles.avatarHolder}>
+                    <img alt="" src={currentUser.avatar} />
+                    <div className={styles.name}>{currentUser.name}</div>
+                    <div>{currentUser.signature}</div>
+                  </div>
+                  <div className={styles.detail}>
+                    <p>
+                      <i className={styles.title} />
+                      {currentUser.title}
+                    </p>
+                    <p>
+                      <i className={styles.group} />
+                      {currentUser.group}
+                    </p>
+                    <p>
+                      <i className={styles.address} />
+                      {currentUser.geographic.province.label}
+                      {currentUser.geographic.city.label}
+                    </p>
+                  </div>
+                  <Divider dashed />
+                  <div className={styles.tags}>
+                    <div className={styles.tagsTitle}>标签</div>
+                    {currentUser.tags.concat(newTags).map(item => (
+                      <Tag key={item.key}>{item.label}</Tag>
+                    ))}
+                    {inputVisible && (
+                      <Input
+                        ref={ref => this.saveInputRef(ref)}
+                        type="text"
+                        size="small"
+                        style={{ width: 78 }}
+                        value={inputValue}
+                        onChange={this.handleInputChange}
+                        onBlur={this.handleInputConfirm}
+                        onPressEnter={this.handleInputConfirm}
+                      />
+                    )}
+                    {!inputVisible && (
+                      <Tag
+                        onClick={this.showInput}
+                        style={{ borderStyle: 'dashed' }}
+                      >
+                        <PlusOutlined />
+                      </Tag>
+                    )}
+                  </div>
+                  <Divider style={{ marginTop: 16 }} dashed />
+                  <div className={styles.team}>
+                    <div className={styles.teamTitle}>团队</div>
+                    <Row gutter={36}>
+                      {currentUser.notice &&
+                        currentUser.notice.map(item => (
+                          <Col key={item.id} lg={24} xl={12}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 12](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)